akka-docs/src/main/paradox/typed/cluster.md

```diff
@@ -95,15 +98,327 @@ Scala
+in the cluster is overloaded, or experiences a long GC pause.
+
+@@@ warning
+
+We recommend against using the auto-down feature of Akka Cluster in production. It
+has multiple undesirable consequences for production systems.
+
+If you are using @ref:[Cluster Singleton](cluster-singleton.md) or @ref:[Cluster Sharding](cluster-sharding.md) it can break the contract provided by 
+those features. Both provide a guarantee that an actor will be unique in a cluster.
+With the auto-down feature enabled, it is possible for multiple independent clusters
+to form (*Split Brain*). When this happens the guaranteed uniqueness will no
+longer be true resulting in undesirable behaviour in the system.
+
+This is even more severe when @ref:[Akka Persistence](persistence.md) is used in
+conjunction with Cluster Sharding. In this case, the lack of unique actors can 
+cause multiple actors to write to the same journal. Akka Persistence operates on a
+single writer principle. Having multiple writers will corrupt the journal
+and make it unusable. 
+
+Finally, even if you don't use features such as Persistence, Sharding, or Singletons, 
+auto-downing can lead the system to form multiple small clusters. These small
+clusters will be independent from each other. They will be unable to communicate
+and as a result you may experience performance degradation. Once this condition
+occurs, it will require manual intervention in order to reform the cluster.
+
+Because of these issues, auto-downing should **never** be used in a production environment.
+
+@@@
+
+### Leaving
+
+There are two ways to remove a member from the cluster.
+
+1. You can stop the actor system (or the JVM process, for example a SIGTERM sent from the environment). It will be detected
+as unreachable and removed after the automatic or manual downing.
+1. A more graceful exit can be performed if you tell the cluster that a node shall leave. 
+This can be performed using @ref:[JMX](../additional/operations.md#jmx) or @ref:[HTTP](../additional/operations.md#http).
+ 
+The @ref:[Coordinated Shutdown](../actors.md#coordinated-shutdown) will automatically run when the cluster node sees itself as
+`Exiting`, i.e. leaving from another node will trigger the shutdown process on the leaving node.
+Tasks for graceful leaving of cluster including graceful shutdown of Cluster Singletons and
+Cluster Sharding are added automatically when Akka Cluster is used, i.e. running the shutdown
+process will also trigger the graceful leaving if it's not already in progress.
+
+Normally this is handled automatically, but in case of network failures during this process it might still
+be necessary to set the node’s status to `Down` in order to complete the removal.
+
+## Serialization 
+ 
+Enable @ref:[serialization](../serialization.md) to send events between ActorSystems and systems
+external to the Cluster. @ref:[Serialization with Jackson](../serialization-jackson.md) is a good choice in many cases, and our
```

Comment:
If it is about sending to "systems external to the cluster" you wouldn't use Akka serialiation but Akka HTTP/spray json, Akka gRPC protobuf etc, so drop that?